akka-cluster-sharding/src/multi-jvm/scala/akka/cluster/sharding/ClusterShardingSpec.scala

```diff
@@ -776,8 +774,8 @@ abstract class ClusterShardingSpec(multiNodeConfig: ClusterShardingSpecConfig)
         val shard = system.actorSelection(lastSender.path.parent)
         val region = system.actorSelection(lastSender.path.parent.parent)
 
-        //Stop the shard cleanly
-        region ! HandOff("1")
+        //Stop the shard cleanly (also clear cache in all nodes)
+        region ! BeginHandOff("1")
         expectMsg(10 seconds, "ShardStopped not received", ShardStopped("1"))
```

Comment:
Strange it this works. `BeginHandOff` should reply with `BeginHandOffAck`.

To simulate real interaction it should send `BeginHandOff` to all regions. Wait for `BeginHandOffAck` from all, and then send `HandOff` and expect `ShardStopped`

Reply:
hmm, that's true. 🤔

Reply:
It doesn't work at all, obviously. 

This is an multi-jvm is not being executed in Travis nor in Jenkins